added smoke tests (#3816)
shopsys/shopsys: docs/storefront/cypress.md

### Commiting flow

By using tools such as those mentioned above, you can simply run cypress tests in the `base` mode, and then check the changed pixels. With this, you avoid the unnecessary step of running the tests in the `actual` mode, but still keep the option of checking exactly which parts of the application have changed.

## How to work with smoke tests

> Smoke tests are a type of testing that quickly verifies the basic functionality of the application. In our codebase, we use smoke tests to ensure that all pages load without errors and meet basic expectations.

Reviewer: ```suggestion
Smoke tests are a type of testing that quickly verifies the application's basic functionality. Our codebase uses smoke tests to ensure that all pages load without errors and meet basic expectations.
```

Author: fixed
